providers/common/securitycheck_fips.c

```diff
@@ -82,7 +82,8 @@ int ossl_fips_ind_digest_check(OSSL_FIPS_IND *ind, int id,
                                OSSL_LIB_CTX *libctx,
                                const EVP_MD *md, const char *desc)
 {
-    int approved = (ossl_digest_get_approved_nid(md) != NID_undef);
+    int nid = ossl_digest_get_approved_nid(md);
```

Comment:
Maybe add a comment for ossl_fips_ind_digest_check() that this function is only called by key exchange currently (so sha1  is not allowed)

Reply:
or change the function name to say similar?

Reply:
renamed function to _digest_exch_check similar to _digest_sign_check just below it.